config/webhook/manifests.yaml

```diff
@@ -89,7 +88,6 @@ webhooks:
 apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
 metadata:
-  creationTimestamp: null
```

Comment:
Based on https://github.com/kubernetes-sigs/controller-tools/issues/402 , `creationTimestamp` is not needed and is fixed by https://github.com/kubernetes-sigs/controller-tools/pull/800 and included in controller tools version 0.14.0.

/lgtm